docs/build/circuit-library.ipynb

```diff
@@ -19,6 +19,57 @@
     "This page lists the different circuit categories the library provides. For a full list of circuits, see the [circuit library API documentation](/api/qiskit/circuit_library)."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0b473350",
+   "metadata": {},
+   "source": [
+    "## Standard gates\n",
+    "\n",
+    "The circuit library also includes standard quantum gates. Some are more fundamental gates (such as the `UGate`), and others are multi-qubit gates that usually need building from single- and two-qubit gates. To add imported gates to your circuit, use the `append` method; the first argument is the gate, and the next argument is a list of qubits to apply the gate to.\n",
+    "\n",
+    "For example, the following code cell creates a circuit with a Hadamard gate and a multi-controlled-X gate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70a7a4df",
```

Comment:
```suggestion
   "id": "a846a845-7ac5-4c92-b124-d2b90a773ba2",
```
I think this should fix CI; the SVG IDs are generated from the cell ID. If not, we can just run the notebook again.